### ISSUE TITLE
Project file contains invalid options

### ISSUE THREAD

**Issue by jamiewinder** [TypeStrong/atom-typescript#423]
No idea what the issue is here, but with the latest version (4.8.1) I continually get an error in Atom stating:

```Error: Project file contains invalid options
    at Object.getProjectSync (C:\Users\jamie.winder\.atom\packages\atom-typescript\dist\main\tsconfig\tsconfig.js:255:32)
    at getOrCreateProjectFile (C:\Users\jamie.winder\.atom\packages\atom-typescript\dist\main\lang\projectCache.js:73:36)
    at Object.getOrCreateProject (C:\Users\jamie.winder\.atom\packages\atom-typescript\dist\main\lang\projectCache.js:138:27)
    at getDiagnositcsByFilePath (C:\Users\jamie.winder\.atom\packages\atom-typescript\dist\main\lang\projectService.js:241:34)
    at Object.errorsForFile (C:\Users\jamie.winder\.atom\packages\atom-typescript\dist\main\lang\projectService.js:259:34)
    at Child.RequesterResponder.processRequest (C:\Users\jamie.winder\.atom\packages\atom-typescript\dist\worker\lib\workerLib.js:49:60)
    at process.<anonymous> (C:\Users\jamie.winder\.atom\packages\atom-typescript\dist\worker\lib\workerLib.js:221:23)
    at emitTwo (events.js:87:13)
    at process.emit (events.js:169:7)
    at handleMessage (child_process.js:306:10)```

All I have to do to make this happen is:
- Create a new folder
- Open folder in Atom
- Create a new TS file

Result: tsconfig.json is auto-created (pasted below), and the error appears.

```
{
    "version": "1.5.0-alpha",
    "compilerOptions": {
        "target": "es5",
        "module": "commonjs",
        "jsx": 2,
        "declaration": false,
        "noImplicitAny": false,
        "removeComments": true,
        "noLib": false,
        "preserveConstEnums": true,
        "suppressImplicitAnyIndexErrors": true
    },
    "filesGlob": [
        "./**/*.ts",
        "./**/*.tsx",
        "!node_modules/**/*.ts",
        "!node_modules/**/*.tsx"
    ],
    "files": [
        "./demo.ts"
    ]
}
```

I'm using Atom 1.0.0.

**Comment by LarsVonQualen** [comment]
+1

**Comment by andresgutierrez** [comment]
+1

**Comment by tae-jun** [comment]
You should try deleting option "jsx" in compilerOptions

**Comment by jamiewinder** [comment]
Deleting the jsx option does the trick... though it's the JSX support I was looking to try out! :)

**Comment by basarat** [comment]
Sorry the default tsconfig creation was broken. Fixed with `4.8.2`
